lib/src/man_up_service.dart

```diff
@@ -3,43 +3,49 @@ part of manup;
 class ManUpService {
   final String url;
   final PackageInfoProvider packageInfoProvider;
-  @visibleForTesting
-  ConfigStorage fileStorage = ConfigStorage();
 
-  String os;
-  Metadata _manupData;
+  final ConfigStorage fileStorage;
+
+  String? os;
```

Comment:
I feel this should never be null - if it's not fed in it should auto-detect from the current platform